tasks/json-eslint.js

```diff
@@ -0,0 +1,69 @@
+/*
+
+    A custom task to validate all JSON(5) as though it were enclosed in a simple javascript fixture.  This does not
+    check for validity, but instead checks things like indentation and other code conventions.
+
+*/
+/* eslint-env node */
+"use strict";
+var fluid  = require("infusion");
+var eslint = require("fluid-eslint");
+
+module.exports = function (grunt) {
+    grunt.registerMultiTask("json-eslint", "Lint JSON(5) files against ESLint rules.", function () {
+        var opts = this.options({});
+
+        if (this.filesSrc.length === 0) {
+            grunt.warn("Could not find any files to validate.");
+            return true;
+        }
+
+        var engine = new eslint.CLIEngine(opts);
+
+        var errorCount = 0;
+        var fileCount  = 0;
+        this.files.forEach(function (f) {
+            var validPaths = f.src.filter(function (filepath) {
+                // Warn on and remove invalid source files (if nonull was set).
+                if (!grunt.file.exists(filepath)) {
+                    grunt.log.warn("Source file '" + filepath + "' not found.");
+                    return false;
+                } else {
+                    return true;
+                }
+            });
+
+            fileCount += validPaths.length;
+
+            fluid.each(validPaths, function (filepath) {
+                try {
+                    var jsonContent       = grunt.file.read(filepath);
+                    var wrappedContent    = fluid.stringTemplate("/* eslint-env node */ \"use strict\"; var wrappedVar = %jsonContent; console.log(wrappedVar);\n", {jsonContent: jsonContent});
```

Comment:
Hoisting this out to a module-scoped constant would improve the structure as well as reduce the line length

Reply:
Yup, done.